.github/workflows/tests.yml

```diff
@@ -41,6 +41,11 @@ jobs:
     - uses: ./.github/actions/setup-test-environment
     - name: Install JavaScript dependencies with Yarn
       run: yarn check || yarn install --frozen-lockfile;
+    - name: Run webpacker setup
+      env:
```

Comment:
I added mysql info here. that should help!

Reply:
Huh. Still didn't work, odd:

```
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:90:in `connect': Access denied for user 'runner'@'localhost' (using password: NO) (Mysql2::Error::ConnectionError)
```

Reply:
I'm going to help with this when I get a moment today...

Browsing the docs turns this up, which seems to be helpful: https://reactjs.org/blog/2013/07/30/use-react-and-jsx-in-ruby-on-rails.html

There's config I could change in `config/environments/development.rb` or perhaps `manifest.json` is committed? I'll have to look deeper.

Reply:
Also this too, I bet config changes need to be made in `config/webpack`:https://stackoverflow.com/a/48769268/10977409

You said that GitHub CI is a `test` environment right?

Reply:
I think i got it. tests are actually running now and failing due to assertions, not setup!

Reply:
yay!!! functional and integration tests passed. i moved the fix to the system tests too so they should be good now as well! 🎉 🎉 🎉

Reply:
@jywarren Nice! Whatever you did, did the trick!!! 🎉 🎈 🍰  I had to restart the CI a few times but currently all tests are passing:

<img width="574" alt="Screen Shot 2021-03-03 at 9 37 20 PM" src="https://user-images.githubusercontent.com/4361605/109916638-d2cc9d00-7c68-11eb-89bb-2ef86279b8ef.png">

And my code is live and working on GitPod!

<img width="697" alt="Screen Shot 2021-03-03 at 9 34 43 PM" src="https://user-images.githubusercontent.com/4361605/109916827-2343fa80-7c69-11eb-864e-9e4580edbd8e.png">

There is an issue with codeclimate:

<img width="830" alt="Screen Shot 2021-03-03 at 9 27 10 PM" src="https://user-images.githubusercontent.com/4361605/109916698-ef68d500-7c68-11eb-9e1b-4be4c81e3802.png">

It mentions `"Can't read property 'recommended' of undefined"`, which I think is caused by these changes I made in `.eslintrc` to install linters for React and React Hooks:

<img width="964" alt="Screen Shot 2021-03-03 at 9 26 19 PM" src="https://user-images.githubusercontent.com/4361605/109916773-0f989400-7c69-11eb-9063-90141032ca45.png">

I think I'm just going to push a commit to `.eslintrc` and see if that gets codeclimate working again, then... maybe this is mergeable?

Reply:
Yeah that does work. codeclimate's displaying errors like normal now after my latest commits. I'll take a look at the errors tomorrow and see what I can address in this PR.

Weird that my changes to `.eslintrc` make ESLint work locally in VSCode, but they break codeclimate. Maybe I can revisit that in another PR.

Reply:
OK that sounds good, thank you! If you can make an issue that'd be great, even a brief one.

Reply:
Will make an issue for getting `.eslintrc` up and running again! I just pushed a rebase for this PR dropping [this commit](https://github.com/publiclab/plots2/commit/2456680208f5b45135d1b5785d8988ee88345e03) completely&mdash;the commit where I installed linters for React and React Hooks. I did this to undo the install to `package.json`.